src/Dropdown.vue

```diff
@@ -73,34 +78,48 @@ export default {
       return this.$parent && (this.$parent.menu || this.$parent.submenu)
     },
     slots () {
-      return this._slotContents
+      return this.$slots.content
```

Comment:
Shouldn't it be `this.$slots.default`?, I don't see a `<slot name="content"></slot>` anywhere